src/components/app/flyouts/settings.js

```diff
@@ -1,27 +1,42 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 import React, { Component } from 'react';
-import {
-  Flyout,
-  FlyoutHeader,
-  FlyoutTitle,
-  FlyoutCloseBtn,
-  FlyoutContent
-} from 'components/shared';
+import Flyout from 'components/shared/flyout';
+
+import './settings.css';
+
+const Section = Flyout.Section;
 
 export class Settings extends Component {
   render() {
-    const { onClose } = this.props;
+    const { t, onClose, theme, changeTheme } = this.props;
+    const nextTheme = theme === 'dark' ? 'light': 'dark';
     return (
-      <Flyout>
-        <FlyoutHeader>
-          <FlyoutTitle>Settings</FlyoutTitle>
-          <FlyoutCloseBtn onClick={onClose} />
-        </FlyoutHeader>
-        <FlyoutContent>
-          Settings
-        </FlyoutContent>
-      </Flyout>
+      <Flyout.Container>
+        <Flyout.Header>
+          <Flyout.Title>{ t('settingsFlyout.title') }</Flyout.Title>
+          <Flyout.CloseBtn onClick={onClose} />
+        </Flyout.Header>
+        <Flyout.Content className="settings-workflow-container">
+
+          <Section.Container collapsable={false} className="app-version">
+            <Section.Header>Version 0.0.1</Section.Header>
```

Comment:
This should be coming from github service call if you are querying latest release from github. Put a TODO